packages/cadl-python/test/generated/dev-driven/resiliency/devdriven/aio/_operations/_operations.py

```diff
@@ -297,11 +257,11 @@ def prepare_request(next_link=None):
             return request
 
         async def extract_data(pipeline_response):
-            deserialized = pipeline_response.http_response.json()
-            list_of_elem = deserialized["value"]
+            deserialized = _deserialize(_models.CustomPageProduct, pipeline_response)
```

Comment:
this won't work because `CustomPageProduct` is not exposed in the `models` folder. This is why I was saying we should switch the import from `from ... import models as _models` to `from ...models import _models` (and do `from ...models import _models_py3 as _models` when necessary in legacy code generation)

Reply:
Got, Thanks for the fix!